### PR TITLE
Make webgl1 greate again

### PR DESCRIPTION
**Pull request by pavletto**
Return webgl1 supporting in globus. Add support in instansing. I'm test it in Firefox with disabled webgl2. Don't fix one momet - something wrong with label `a_rotation` in `label` vertexShader. @Zemledelec can you check it please?

### CODE REVIEW

**Comment by yeluoyunfei** [comment]
@pavletto  Just saw this pull request, that's great! is the change ok to merge?

**Comment by pavletto** [comment]
@Zemledelec ping

**Comment by Zemledelec** [comment]
> @Zemledelec ping

testing

File: src/og/webgl/Handler.js [hunk 1]
```diff
@@ -971,6 +973,14 @@ class Handler {
         });
     }
 
+    /**
+     * Check is gl context type equals webgl2
+     * @public
+     */
+    isWebGl2() {
```
Comment (Zemledelec) [inline src/og/webgl/Handler.js]: Better to make property isWebgl2 right after initialization, and get rid of the method.

File: src/og/webgl/Handler.js [hunk 2]
```diff
@@ -980,10 +990,12 @@ class Handler {
     createDefaultTexture(params, success) {
         let imgCnv;
         let texture;
+        const is2 = this.isWebGl2();
+
         if (params && params.color) {
             imgCnv = new ImageCanvas(2, 2);
             imgCnv.fillColor(params.color);
-            texture = this.createTexture_n_webgl2(imgCnv._canvas);
+            texture = is2 ? this.createTexture_n_webgl2(imgCnv._canvas) : this.createTexture_n_webgl1(imgCnv._canvas);
```
Comment (Zemledelec) [inline src/og/webgl/Handler.js]: Function callbacks should be createTexture_n which assigned during webgl context initialization : createTexture_n = createTexture_n_webgl2

**Review by Zemledelec — CHANGES_REQUESTED** [review]

File: src/og/entity/GeoObjectHandler.js [hunk 3]
```diff
@@ -404,7 +404,7 @@ class GeoObjectHandler {
             gl.bindBuffer(gl.ELEMENT_ARRAY_BUFFER, this._indicesBuffer[ti]);
 
             if (tagData.iCounts) {
-                gl.drawElementsInstanced(
+                p.drawElementsInstanced(
```
Comment (Zemledelec) [inline src/og/entity/GeoObjectHandler.js:407]: Better to get rid of if case during webgl context initialization.

**Review by Zemledelec — CHANGES_REQUESTED** [review]

**Comment by Zemledelec** [comment]
need refactor for Program.js
```
    drawElementsInstanced(...args) {
        const gl = this.gl;
        gl.drawElementsInstanced ? gl.drawElementsInstanced(...args) : gl.getExtension('ANGLE_instanced_arrays').drawElementsInstancedANGLE(...args);
    }
```

The idea is not using if case and function call. Instead assign a method to a this.drawElementsInstanced during initialization right inside program

**Comment by pavletto** [comment]
Check!

**Review by Zemledelec — APPROVED** [review]

**Comment by Zemledelec** [comment]
Do not forget. I've checked up.

INSERT A SMALL DESCRIPTION OF WHAT YOU CHANGED HERE

Verify that...
 npm test passes
 npm run api passes
 npm run lint passes
 npm run build still works

**Pull request merged**